### ISSUE TITLE
can not reproduce experiment shown in figure

### ISSUE THREAD

**Issue by zcyang** [devsisters/DQN-tensorflow#17]
Hi, can you share a configuration that can reproduce the results you showed on the figure?
I run the default M1 configuration and only get average episodic reward at around 3.

I tried to change the configurations like setting action_repeat = 4, change learning_rate, add double_q and duel_q, there is no much change.

Many thanks!

**Comment by carpedm20** [comment]
I think commit before Dec 2016 cause some problem. I'll dig into this and this is highly related to #21 .

**Comment by zcyang** [comment]
any update on this?
